privatebin/templates/statefulset.yaml

```
@@ -0,0 +1,71 @@
+{{- if or (eq .Values.controller.kind "StatefulSet") (eq .Values.controller.kind "Both") }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "privatebin.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "privatebin.name" . }}
+    helm.sh/chart: {{ include "privatebin.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "privatebin.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "privatebin.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: configs
+              mountPath: /srv/cfg
+            - name: storage
+              mountPath: {{ .Values.datapath }}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+    {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      volumes:
+        - name: configs
+          configMap:
+            name: {{ include "privatebin.fullname" . }}-configs
+  volumeClaimTemplates:
+  - metadata:
+      name: storage
+    spec:
+      accessModes: [ "ReadWriteMany" ]
+      storageClassName: {{ .Values.controller.pvc.storageClass  }}
```

This is to keep consistent with how other charts manage persistent volumes

```suggestion
    {{- if .Values.controller.pvc.storageClass }}
      {{- if (eq "-" .Values.controller.pvc.storageClass) }}
      storageClassName: ""
      {{- else }}
      storageClassName: "{{ .Values.controller.pvc.storageClass }}"
      {{- end }}
    {{- end }}
```